sections/introduction.tex

```diff
@@ -2,18 +2,18 @@ \chapter{Introduction} % (fold)
 \label{cha:introduction}
 Ever since the dawn of mathematics, mathematicians has strived for faster computations while maintaining correctness.
 In our time computers have become the main target for doing extensive computations, and their CPUs are what the majority of programming languages apply to.
-CPUs are good at handling sequential instructions and moderns CPUs even exploit their multiple cores in order to attend to serial and task parallel workloads.
+CPUs are good at handling sequential instructions and modern CPUs even exploit their multiple cores in order to attend to serial and task parallel workloads.
 As applications and computations grow larger and more complex, programmers and mathematicians find themselves in need of more and more computation power.
-At the moment this demand is met mostly with faster multicored CPUs however Moore's law, which predicts an exponential growth in transistor count, thus faster computations on said CPUs, seems to be a trend coming to an end.
-Experts foresee this stagnation of Moore's Law as soon as 2020 or 2022; hence other options than CPUs need to be explored.\citep{Moore2013}
+At the moment this demand is met mostly with faster multi-cored CPUs however Moore's law, which predicts an exponential growth in transistor count, thus faster computations on said CPUs, seems to be a trend coming to an end.
+Experts foresee this stagnation of Moore's Law as soon as 2020 or 2022; hence other options than CPUs need to be explored. \citep{Moore2013}
 
 One of these options is the GPU, or graphics processing unit, which mainly is used to generate the graphics that are displayed on the screen.
 The GPU is specialised in performing a vast number of smaller computations in parallel, since that essentially is what generating graphics is about.
-Because of this specialisation, the GPU can handle data parallel workload just as well as the CPU can handle task parallel workloads.
-Mathematicians and programmers alike can utilise this advantage and hereby complete large computations on big sets of date in parallel.
+Because of this specialisation, the GPU can handle data parallel workloads just as well as the CPU can handle task parallel workloads.
+Mathematicians and programmers alike can utilise this advantage and hereby complete large computations on big sets of data in parallel.
 The concept of executing computations on the GPU not related to graphics, is commonly know as GPGPU or General-purpose computing on graphics processing units. 
 
-The following sections will go into further detail regarding both the architecture of the GPU but also existing mathematic programming languages.
+The following sections will go into further detail regarding both the architecture of the GPU but also existing mathematical programming languages.
 Furthermore it will be described how one can target the GPU for computational purposes, and the different opportunities for doing so. 
 
```

Comment:
Kan ikke finde ud af om der måske skal være nogle spørgsmål som vi vil undersøge her?

som f.eks:
How does the GPU do these parallel calculations, and why is it better at it than the CPU?
How can a programmer utilise the functionality found in the GPU, and what problems might arise from doing so?